src/fundus/publishers/de/heise.py

```diff
@@ -30,7 +30,18 @@ class V1(BaseParser):
             " or @class='antwort rte__abs--antwort'"
             " or @class='frage rte__abs--frage'"
             " or @class='json-ld-paid-content-marker'] "
-            " | //div[@class='article-layout__content article-content']//ul["
+            " | //div[contains(class, article-content)]//ul["
+            "@class='rte__list rte__list--unordered' or @class='boxtext']/li | "
+            "//div[@class='ringCommonDetail ringBlockType-paragraph ']/p[not(@class"
+            " or ((string-length(text()) < 3) and (contains(text(), '(') or contains(span, '(')))"
+            " or contains(text(), '=== Anzeige / Sponsorenhinweis')"
+            " or contains(text(), 'Tipp: Wir sind bei WhatsApp!')"
+            " or contains(a, 'heise+ abonnieren')"
+            " or contains(text(), '► '))"
+            " or @class='antwort rte__abs--antwort'"
+            " or @class='frage rte__abs--frage'"
+            " or @class='json-ld-paid-content-marker'] "
+            " | //div[@class='ringCommonDetail ringBlockType-paragraph ']//ul["
```

Comment:
Is this all `techstage` related?

Reply:
Yes, they use a slightly different formatting.

Reply:
But I did realise that the selectors can be simplified, as techstage does not seem to contain as much bloat as heise